orange/src/test/java/com/hartwig/hmftools/orange/algo/linx/LinxReportableClustersTest.java

```diff
@@ -24,13 +24,49 @@ public class LinxReportableClustersTest
     @Test
     public void shouldFindAllReportableClusters()
     {
-        LinxData linxData = createTestLinxData();
+        LinxData linxData = linxDataBuilder().build();
         Set<Integer> clusters = LinxReportableClusters.findReportableClusters(linxData);
         assertEquals(new HashSet<>(Set.of(10, 50, 100)), clusters);
     }
 
+    @Test
+    public void shouldNotCountSimpleClusters()
+    {
+        // cluster 10 has single chain link and no exons -> should not be counted
+        LinxData linxData = linxDataBuilder()
+                .clusterIdToChainCount(Map.of(10, 1, 50, 2, 100, 2))
+                .clusterIdToExonCount(Map.of(50, 1, 100, 1))
+                .build();
+        Set<Integer> clusters = LinxReportableClusters.findReportableClusters(linxData);
+        assertEquals(new HashSet<>(Set.of(50, 100)), clusters);
+    }
+
+    @Test
+    public void shouldCountMultiLinkClustersWithNoExon()
+    {
+        // cluster 10 has two chain links and no exons -> not simple so should be counted
+        LinxData linxData2 = linxDataBuilder()
```

Comment:
I wouldn't use "2" in the name, since the test is independent from 1 an 3. Either just `linxData` or a more descriptive name

Reply:
yeah I think I had these all in a single test before splitting up, and didn't renamed the vars.